src/main/java/address/updater/UpdateManager.java

```diff
@@ -107,6 +109,7 @@ private void checkForUpdate() {
         if (latestVersion.get().isEarlyAccess() != MainApp.IS_EARLY_ACCESS) {
             raise(new UpdaterFinishedEvent(MSG_DIFF_CHANNEL));
             logger.debug(MSG_DIFF_CHANNEL);
+            //TODO: how could this happen?
```

Comment:
This should never happen unless we mess up the update data. `assert false` seems to be too harsh as the chance that we might mess up the update data during release.